docs/marketing/brand-package-prompt.md

```
@@ -0,0 +1,268 @@
+- Display 2xl: 96px
+- Display xl: 72px
+- Display lg: 56px
+- Display md: 40px
+- Body: 16px
+- Mono sm: 13px
+
+### 3.4 Logo mark
+
+The `BrandMark` is three strokes drawn in SVG inside a 100x100 viewBox:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)